packages/stores/types.ts

```diff
@@ -288,11 +292,21 @@ type TTradersHubStore = {
     selectRegion: (region: string) => void;
     is_real_wallets_upgrade_on: boolean;
     toggleWalletsUpgrade: (value: boolean) => void;
+    openFailedVerificationModal: (selected_account_type: Record<string, any>) => void;
     financial_restricted_countries: boolean;
     selected_account_type: string;
     no_CR_account: boolean;
     no_MF_account: boolean;
-    setTogglePlatformType: () => void;
+    setTogglePlatformType: (value: string) => void;
+    is_tour_open: boolean;
+    selected_platform_type: string;
+    // TODO: must be BrandConfig type but can't import it from appstore. Maybe we need to create the same type in 'stores' package. Have to think about this!
```

Comment:
@sergei-deriv Yes please create the type here, Then appstore can use it.

Reply:
@farzin-deriv thanks for clarification because I was not sure what is a right answer in this case